spec/features/pom_staff_female_feature_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+feature "get poms list" do
+  before do
+    signin_spo_user
+
+    # mock to ensure we are in a women's prison
+    allow(PrisonService).to receive(:womens_prison?).and_return(true)
+    visit prison_poms_path('LEI')
+  end
+
+  context 'with active poms' do
+    it 'shows the page', vcr: { cassette_name: 'prison_api/pom_staff_female_active_list' } do
+      expect(page).to have_content("Manage your staff")
+      expect(page).to have_content("Active Probation officer POM")
+      expect(page).to have_content("Inactive staff")
+      expect(page).to have_content("High complexity cases")
+    end
+
+    it 'can view a POM', vcr: { cassette_name: 'prison_api/pom_staff_female_view_pom' } do
+      # click on the first offender
+      within "#active_probation_poms" do
+        first('td.govuk-table__cell > a').click
+      end
+
+      expect(page).to have_css(".govuk-button", count: 1)
+      expect(page).to have_content('POM level')
+      expect(page).to have_content('Working pattern')
+      expect(page).to have_content('Status')
+    end
+  end
+
```

Comment:
This feels much more like a view test than a feature test, and the assertions feel a bit weak